### ISSUE TITLE
Create dashboard for returning users

### ISSUE THREAD

**Issue by clash99** [Cadasta/cadasta-platform#1069]
I thought we had a backlog task for this but I think most of the tasks associated are for the unregistered user's "home" page.  

After conversations with @wonderchook, @MappingKat , and others, I think we need to start thinking of creating a better dashboard for returning users.  Some ideas to include:

- Updates/ stats on projects user belongs to apart from other projects (currently there isn't a way for users to know which projects they belong to from home or project index)
- Roles/permissions for each project (currently there isn't a way for users to know their roles/permission level)
- Messaging regarding recent releases, ui changes, upcoming events
- Other ideas?

**Comment by wonderchook** [comment]
Indicate what projects have been updated since their last visit. e.g. if new data has been added

**Comment by dpalomino** [comment]
Yes, I think this is linked to the User Dashboard feature (id 44.00 in the backlog). Requirements are still not developed, but I was thinking to include information like: 
- Organizations information: which orgs they belong to, role (admin/member)
- Projects information: which projects they belong to, role (Project Manager, Data Collector, etc), stats for this projects. (BTW, @wonderchook good idea to include the last updated timestamp, I've added it to the backlog)
- Saved search queries (when available)
- Links to custom reports (when available)
- Bookmarks (to specific records?)
- Activity log of the user

@clash99, I've also added the messaging about recent releases, upcoming events, news, etc. That's a great idea.

**Comment by dpalomino** [comment]
Also related with #994.

**Comment by MappingKat** [comment]
I think that the current home dashboard is great for Cadasta admin (us).  For partners and organizations, it would be nice to the elements that you all have mentioned.  (For releases and updates, we can use a chat messenger to announce those.)

![existing users](https://cloud.githubusercontent.com/assets/1077436/22254397/b8379382-e211-11e6-9ea9-90b740d5ff18.png)

I think the dashboard should focus on the details of the projects--- although it would range depending on the users' role.  

*New Users*
It would be also beneficial to create a more friendly dashboard for new users.  Instead of overwhelming new users with all of the test projects, we narrow their focus on one or two demo projects that are complete.  That way new users have a full understanding of the potential of the Cadasta platform. 

![new users](https://cloud.githubusercontent.com/assets/1077436/22254381/aa5bbb44-e211-11e6-91d5-5de5d695ebca.png)

**Comment by clash99** [comment]
@dpalomino - In talking with @alukach today, he mentioned needing a place for users to see status or possibly link to background processes (think importing or exporting of large files).  I think the user dashboard and possibly the project dashboard may be the right place to include this.  Just wanted to drop you a quick note since we mentioned trying to better define user dashboard requirements this week in prep for intern.

**Comment by dpalomino** [comment]
Thanks @clash99! Yes I agree, thanks for the heads-up!

**Comment by clash99** [comment]
With the user dashboard outreachy internship starting, I've created an epic #1491 with tasks for the items listed here for returning users.  I'm modifiying this task to pertain to the new user possibilities.

**Comment by clash99** [comment]
We should also add some marketing text to the project overview pages. For unregistered users right now we only present a "sign in" option but we could do a better job here telling new users about benefits of registering, etc.

![screenshot 2017-05-23 09 50 18](https://cloud.githubusercontent.com/assets/1119621/26360349/81b32396-3f9d-11e7-9bde-7e2019cd96b9.png)

**Comment by dpalomino** [comment]
@clash99. About the "marketing language" for promoting users to register, what about something like: 

Haven't you registered yet? Cadasta platform will help you to document your land and resource rights, share the information as your convenience, or ask to join one of the many ongoing projects that are already documenting their rights on the ground. Please, register and take a look!

@bethschechter, any ideas?

**Comment by clash99** [comment]
I've updated it to break it more into two halves - one for new registrations and one for sign-ins. I think this is in good shape for launch but we should add this to the list for improving our marketing messages.

![screenshot 2017-05-24 19 45 43](https://cloud.githubusercontent.com/assets/1119621/26431402/0a74a5f2-40ba-11e7-9a64-028f780fa0a5.png)

**Comment by dpalomino** [comment]
@bethschechter, any suggestions for the marketing messages? 

My two cents: 
- I'd avoid to use "cloud" word. Some partners are still some kind scared with that
- For the new users, I'd also mention geometries, something like "to collect, store and manage geometries, field data and documents related to land rights in our secure platform". 

Thanks @clash99!

**Comment by bethschechter** [comment]
@dpalomino I agree with you on the cloud language. Also, in general, I'd like to see less text in both of those squares. Let me see if I can distill it down...

**Comment by clash99** [comment]
@dpalomino and @bethschechter - i'm working on this now so if you can provide something soon, I can include it in the PR

**Comment by bethschechter** [comment]
kk working on it now

**Comment by bethschechter** [comment]
@dpalomino @clash99 how about this:

New Users

Register for free to securely collect, manage, and store your land and resource rights documentation, geometries, and other data. 

[Register]

Have An Account?

Sign in to manage your projects and access permissions-based data.

[Sign in]

**Comment by bethschechter** [comment]
I went with shorter rather than longer but tried to get some of those suggestions in there, David.

**Comment by clash99** [comment]
Thanks @bethschechter.  I've updated the copy on the project overview page.

<img width="752" alt="screenshot 2017-05-30 17 50 53" src="https://cloud.githubusercontent.com/assets/1119621/26608316/8f096cbc-4560-11e7-9d13-a57eb62752e1.png">

**Comment by bethschechter** [comment]
Nice, tho it would be great if I could get that first one down to 2
lines...hmmm...

**Comment by clash99** [comment]
If you are happy with the text, I can adjust the layout to line up the buttons. Its responsive so the number of lines will change based on screen resolution.

**Comment by bethschechter** [comment]
oooh that would be cool. Thank you!
